include/freerdp/settings.h

```diff
@@ -822,301 +822,301 @@ typedef struct _RDPDR_PARALLEL RDPDR_PARALLEL;
 struct rdp_settings
 {
 	/**
-	 * WARNING: this data structure is carefully padded for ABI stability!
```

Comment:
Wrong indentation, we use tabs

Reply:
Oops, I relied on autoformat, but of course it doesn't process the scripts. Will do.

Reply:
Well, without reformating of the #define lines, they're not all aligned, so it's harder to read.
Now, the script can be improved to use tabs instead of spaces, but it's harder than just using printf %-50s.  If you insist on this part, I can do it still.